applications/ShapeOptimizationApplication/tests/.gitignore

```diff
@@ -0,0 +1,7 @@
+**/Optimization_Results/
```

Comment:
Is there a reason why you deleted the "delete"-command from the test runs and added the output to the gitignore? I personally find it much cleaner to delete all unnecessary output instead of ignoring it.

Reply:
I often use tests in order to compare new developments with the "tested" stuff. The fact that the results are deleted after a successful run is annoying. IMO deleting the results after the test is doing extra stuff that is not part of the default behavior of the code. But this may be my personal preference.